lib/polyfill/patchedmediakeys_apple.js

```diff
@@ -163,6 +169,23 @@ shaka.polyfill.PatchedMediaKeysApple = class {
 
     // Convert the prefixed init data to match the native 'encrypted' event.
     const uint8 = shaka.util.BufferUtils.toUint8(event.initData);
+
+    try {
+      const base64 = shaka.util.StringUtils.fromCharCode(uint8);
+      const data = JSON.parse(base64);
```

Comment:
You call it `base64`, but then treat it as a JSON string.  Perhaps that's the wrong name for it?